stregsystem/templates/stregsystem/base.html

```diff
@@ -0,0 +1,13 @@
+<html>
+    <head>
+        <title>
+            {% block title %}{% endblock %}
```

Comment:
You can have a default title, e.g. {% block title %}Stregsystem{% endblock %}

This works a lot nicer, since you can then overwrite it from templates that extends this one, if need be, but still always have a default.